### DISCUSSION TITLE
MVP从1倍速调整倍速时会稳定卡顿

### DISCUSSION THREAD

**Discussion by CloudThick** [hooke007/mpv_PlayKit#525] [Q&A 问答]
MPV在播放视频时调整倍速 以1倍速为起点或以1倍速为终点，在到达1倍速时会卡顿一下，大概0.5秒的样子，非常稳定。

比如：
从0.9倍速调到1.6倍速时，没有任何问题，非常丝滑没有卡顿
但从1倍速调到1.6倍速时，视频会卡半秒钟，之后再进行调速，从1.6调回来时也会在1倍速那里卡半秒

这是MPV自身的缺陷吗？还是别的原因，应该不是配置问题，因为只有1倍速是这样。

**Comment by hooke007** [comment]
你说的的是lazy还是无配置的mpv
如果是lazy那大概率只是在切换同步模式
https://github.com/hooke007/MPV_lazy/blob/b6b10a53fe86297ed0eef555172331b24bedcd08/portable_config/profiles.conf#L27

另外一提，我没有这个问题，没见过有人提过类似问题。

**Reply by CloudThick** [reply]
感谢提醒，我把原来的MPV卸载重装了一下后发现这个问题已经不存在了，应该是我不懂瞎改了某个设置😰导致的这个问题。
现在一切正常播放了，谢谢作者大大，你真是个好人，给你添麻烦了。🤧
